src/app/helptext/storage/volumes/manager/manager.ts

```diff
@@ -21,7 +21,7 @@ manager_disknumExtendConfirmMessage : T("It is not recommended to extend a pool
 
 manager_vdevtypeErrorMessage : T("Adding data vdevs of different types is not supported."),
 
-manager_stripeVdevTypeErrorMessage : T("vdev is highly discouraged and will result in data loss if it fails"),
+manager_stripeVdevTypeErrorMessage : T("vdev may result in data loss if it fails combined with a power outage."),
```

Comment:
small rework suggestion to put the danger first.
```suggestion
manager_stripeVdevTypeErrorMessage : T("Data loss can occur when the vdev fails and the system encounters a power outage!"),
```

Reply:
> There are two things to consider.
> 
> For data vdev the old text should remain with the same color.
> 
> The new text and color should be used for Log vdev only.

If we add a new independent warning message that appears on `log vdev` type and can be visible with the existing message at the same time. For example, Data Vdev has 1 disk and we will see the red message then Log Vdev has 1 disk and we will see the orange message like this, does it look good enough?:
<img width="1440" alt="Screenshot 2021-04-01 at 14 16 56" src="https://user-images.githubusercontent.com/351613/113287363-2973d800-92f6-11eb-8046-71df9f2b2ed3.png">

Reply:
@Mrt134 looks good, thanks for the suggestion. one thing that I wondering it is just a part of the warning message. Full message would be `A stripe log vdev may result in data loss if it fails combined with a power outage.` and `A stripe {type} vdev is highly discouraged and will result in data loss if it fails` like on the screenshot above.

Reply:
LGTM

Reply:
@Mrt134 there are two similar messages now. Could you take a look, please?